src/datastore/postgres-store.ts

```diff
@@ -1217,6 +1219,9 @@ export class PgDataStore
             await this.updateNamespaces(client, entry.tx, namespace);
           }
         }
+        if (newNftEvents) {
```

Comment:
If possible we should skip this step during event-import (otherwise it would be slowed down dramatically), and only perform the refresh at the end of the replay.

Reply:
Done @zone117x LMK what you think about this approach